pavelib/assets.py

```diff
@@ -51,16 +57,39 @@ def configure_paths():
         lms_css = theme_dir / "lms" / "static" / "css"
         if lms_sass.isdir():
             lms_css.mkdir_p()
-            SASS_DIRS.append(lms_sass)
+            THEME_SASS_DIRECTORIES.append(lms_sass)
         cms_sass = theme_dir / "cms" / "static" / "sass"
         cms_css = theme_dir / "cms" / "static" / "css"
         if cms_sass.isdir():
             cms_css.mkdir_p()
-            SASS_DIRS.append(cms_sass)
+            THEME_SASS_DIRECTORIES.append(cms_sass)
 
 configure_paths()
 
 
+def applicable_sass_directories(systems=None):
+    """
+    Determine the applicable set of SASS directories to be
+    compiled for the specified list of systems.
+
+    Args:
+        systems: A list of systems (defaults to all)
+
+    Returns:
+        A list of SASS directories to be compiled.
+    """
+    if not systems:
+        systems = ALL_SYSTEMS
```

Comment:
Sorry I am jumping into this PR very late in the process, so you can ignore me, but just wondering if we change the method signature from "systems=None" to "systems=ALL_SYSTEMS" ?